product_unique_serial/README.md

```diff
@@ -0,0 +1,4 @@
+Product Unique Serial Number
```

Comment:
Please put description in a README.rst file. (odoo only understand rst format and github will not understand rst text in md file)

Plus, use OCA template:
https://raw.githubusercontent.com/OCA/maintainers-tools/master/template/module/README.rst